### DISCUSSION TITLE
http: TLS handshake error from client:57796: EOF

### DISCUSSION THREAD

**Discussion by aoberest** [seaweedfs/seaweedfs#2611] [Q&A]
Hello,
I often get a message from filer http: TLS handshake error from client:57796: EOF

My clients is java application in k8s cluster. 

Could you tell me, please, in what cases this message can appear?

**Comment by chrislusf** [comment]
Maybe needs to enable tcp keep alive https://stackoverflow.com/questions/34823724/golang-tls-handshake-error

**Reply by aoberest** [reply]
Thanks for answer. I will check it.
